Dockerfile.production

```
@@ -18,10 +18,33 @@ RUN --mount=type=cache,target=/usr/local/cargo/registry \
 
 WORKDIR /app
+# COPY --chown=nonroot:nonroot ./config /app/config
 COPY --from=base --chown=nonroot:nonroot /usr/app/bin/openzeppelin-relayer /app/openzeppelin-relayer
 COPY --from=base /usr/lib/libssl.so.3 /usr/lib/libssl.so.3
 COPY --from=base /usr/lib/libcrypto.so.3 /usr/lib/libcrypto.so.3
 
+ARG CONFIG_DIR=
+ARG NETWORKS_DIR=
+ARG LAUNCHTUBE_DIR=
+
+RUN --mount=type=bind,src=${CONFIG_DIR},target=/mnt/config,ro,required=false \
```

Hadolint (2.12.0)

[error] unexpected "required=fal"
expecting "from=", "readonly", "readonly=", "readwrite", "readwrite=", "ro", "ro=", "rw", "rw=", source=, or target=

(DL1000)